src/__tests__/app.test.js

```diff
@@ -0,0 +1,154 @@
+import * as app from '../store/app';
+
+/**
+ * MOCK DATA
+ */
+const appWaitingState = {
+  app: {
+    appWaiting: 1,
+  }
+};
+
+const mobileDrawerOpenedState = {
+  app: {
+    isMobileDrawerOpened: true
+  }
+};
+
+const multipleAlertsState = {
+  app: {
+    alerts: [
+      { desc: 'firstAlert', severity: 'error', title: 'firstAlert' },
+      { desc: 'lastAlert', severity: 'error', title: 'lastAlert' }
+    ],
+  }
+};
+const alert = {
+  desc: 'firstAlert',
+  severity: 'error',
+  title: 'firstAlert'
+};
+const alerts = [
+  { desc: 'firstAlert', severity: 'error', title: 'firstAlert' },
+  { desc: 'lastAlert', severity: 'error', title: 'lastAlert' }
+];
+const lastAlert = {
+  id: 1,
+  desc: 'lastAlert',
+  severity: 'error',
+  title: 'lastAlert'
+};
+
+const removeAlertState = {
+  alerts: [{ desc: 'api success desc', severity: 'error', title: 'API ERROR' }],
+  appWaiting: 0,
+  isMobileDrawerOpened: false
+};
+const appState = {
+  alerts: [],
+  appWaiting: 1,
+  isMobileDrawerOpened: false
+};
+/**
+ * ACTIONS
+ */
+describe('actions', () => {
+  it('toggleMobileDrawer', () => {
+    const expectedAction = {
+      type: app.TOGGLE_MOBILE_DRAWER,
+    };
+    expect(app.toggleMobileDrawer()).toEqual(expectedAction);
+  });
+  it('setAppWaiting', () => {
+    const expectedAction = {
+      type: app.SET_APP_WAITING,
+    };
```

Comment:
The payload is missing from this. That could be true or false. I mean the undefines is still something that can occure, however we need to test the other possibilities as well.

Reply:
> In general if you put a store test file into the **test** folder then put them inside a store folder as well. In the future we can identify them more easliy if we add more tests or ui tests as well.

sure